mediapipe/graphs/instantmotiontracking/calculators/gl_animation_overlay_calculator.cc

```diff
@@ -159,17 +159,7 @@ class GlAnimationOverlayCalculator : public CalculatorBase {
   float perspective_matrix_[kNumMatrixEntries];
 
   // Used to normalize a 3-element float array (vertex normal)
-  void Normalize3f(float input[3]) {
-    float product = 0.0;
-    product += input[0] * input[0];
-    product += input[1] * input[1];
-    product += input[2] * input[2];
-    float magnitude = sqrt(product);
-    input[0] /= magnitude;
-    input[1] /= magnitude;
-    input[2] /= magnitude;
-  }
-
+  void Normalize3f(float input[3]);
```

Comment:
super-nit: For reference, this function doesn't actually have anything to do with our calculator, so we could make it static in an anonymous namespace (or at least const- qualify it).  But no need to make those changes now.